runtime/CMakeLists.txt

```diff
@@ -36,3 +36,7 @@ install(
   DESTINATION lib/kllvm/rust/deps
   PATTERN *.rlib
 )
+
+if (CMAKE_BUILD_TYPE STREQUAL "Release")
+  install(SCRIPT ../cmake/mkrlib.cmake)
```

Comment:
I don't understand the CMake here. Is this adding additional build steps or trying to change dependencies? Is this the right way to do this?

Reply:
This is adding a script that runs when the code is installed. That script proceeds to modify the rlib files installed from cargo so that they are suitable for lto. I did it this way because it's easier to get the list of all the libraries that need to be munged at install time after they've been created than trying to statically list all the recursive rust dependencies of our rust code.

Reply:
It's also much more stable because if we depend on a rust package and that package adds new dependencies in a new version, we don't have to explicitly change the cmake script.

Reply:
Ok, running this at installation time sounds pretty reliable. I was worried about interactions with compilation if it ran during the build.